ext/mbstring/libmbfl/mbfl/mbfl_convert.c

```diff
@@ -190,11 +189,6 @@ void mbfl_convert_filter_reset(mbfl_convert_filter *filter, const mbfl_encoding
 
 void mbfl_convert_filter_copy(mbfl_convert_filter *src, mbfl_convert_filter *dest)
 {
-	if (src->filter_copy != NULL) {
-		src->filter_copy(src, dest);
-		return;
```

Comment:
In the commit message you say that the result of the copy gets overwritten by the bitwise copy -- but we have a return here, so that doesn't sound right?

Reply:
Hmm. I wonder what I was thinking there. You're totally right.

Reply:
Thinking about what to do here. Maybe I should just back out of this change for now.